tensorflow/c/eager/BUILD

```diff
@@ -829,6 +829,7 @@ tf_cuda_cc_test(
     args = ["--heap_check=local"],
     tags = [
         "no_windows",
+        "no_rocm",
```

Comment:
@stevenireeves 

did you mean to put the `no_rocm` tag on this one?

based on the logs from
* http://ml-ci.amd.com:21096/job/develop-upstream-unit-tests/job/tensorflow-upstream-unit-tests-cpu/123/console

the failure in `Ubuntu-CPU` was this one
```
//tensorflow/python:function_test                                        FAILED in 3 out of 3 in 64.4s
```

Reply:
@deven-amd That was a mistake, that was the test right before function:test. Must have misread.